recipes/hsm/all/conanfile.py

```diff
@@ -0,0 +1,48 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+import os
+
+required_conan_version = ">=1.32.1"
+
+class HsmConan(ConanFile):
+    name = "hsm"
+    license = "MIT"
+    homepage = "https://github.com/erikzenker/hsm.git"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "The hana state machine (hsm) is a finite state machine library based on the boost hana meta programming library. It follows the principles of the boost msm and boost sml libraries, but tries to reduce own complex meta programming code to a minimum."
+    topics = ("state machine", "template meta programming")
+    requires = "boost/1.72.0"
```

Comment:
```suggestion
    requires = "boost/1.76.0"
```

Is it Boost 1.72 really required? Otherwise, use the latest version to keep aligned with other packages using Boost.

Reply:
77 is out as well

Reply:
77 ?? 

![jumanji-adventure](https://user-images.githubusercontent.com/4870173/135079071-cca0df47-0fd4-476f-b5a1-b0d7c015e83b.gif)